main/acle.md

```diff
@@ -1948,6 +1968,20 @@ See [Half-precision brain
 floating-point](#half-precision-brain-floating-point) for details
 of half-precision brain floating-point types.
 
+#### Non-widening brain 16-bit floating-point support
```

Comment:
In the Arm ARM March 2024, the feature name is `FEAT_SVE_B16B16`, which implies `FEAT_SME2` or `FEAT_SVE2`. Differences with the proposed spec are:
 * spelling of the feature name
 * it does not imply SVE2.1 or SME2.1 as the text seems to suggest

 Perhaps two separate macros are not needed and equivalent functionality can be obtained by, e.g.
```
#if defined (__ARM_FEATURE_SME) && defined (__ARM_FEATURE_SVE_B16B16)
...
```

Reply:
Yeah, the PR was based on an earlier version of the spec.  I've now updated it for the 2024-06 version.